src/main/java/core/basesyntax/utils/generate/impl/csv/CsvGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.utils.generate.impl.csv;
+
+import core.basesyntax.storage.impl.StorageImpl;
+import core.basesyntax.utils.generate.ReportGenerator;
+
+public class CsvGeneratorImpl implements ReportGenerator {
+    private static final String LINE_SEPARATOR = ",";
+    private static final String REPORT_INIT = "fruit,quantity";
+
+    @Override
+    public String getReport(StorageImpl storage) {
```

Comment:
According to the encapsulation principle, the method `getReport` should not directly depend on the `StorageImpl` class. Consider using an interface to decouple the `CsvGeneratorImpl` from a specific implementation of storage.